src/index.js

```diff
@@ -52,29 +46,83 @@ inquirer
       message: "Enter your Axe Monitor password:",
     },
     {
-      type: "confirm",
-      name: "limit",
-      message: "Limit to specific month?",
-      default: true,
-    },
-    {
-      type: "input",
-      name: "date",
-      message: "Enter MM/YYYY to limit report:",
-      default: `${month}/${year}`,
-      validate: (input) => {
-        return new Promise((resolve, reject) => {
-          if (/^(0?[1-9]|1[012])[\/\-]\d{4}$/.test(input)) {
-            resolve(true);
-          }
-          reject("Incorrect date format");
-        });
-      },
-      when: (answers) => answers.limit,
+      type: "list",
+      name: "path",
+      message: "What are you trying to do today?",
+      choices: [
+        {
+          name: "projects",
+          value: "projects",
+          description: "Export data about my favorited projects.",
+        },
+        {
+          name: "issues",
+          value: "issues",
+          description: "Export all of the issues from a specific project.",
+        },
+      ],
     },
   ])
   .then((answers) => {
-    reporter(answers);
+    if (answers.path === "issues") {
+      // Prompt for the project ID and then export all issues.
+      inquirer
+        .prompt([
+          {
+            type: "input",
+            name: "projectid",
+            message:
+              "What are the Project Ids (the *** in {Axe Monitor URL}/worldspace/issues/*** separated by commas(,))?",
```

Comment:
Could we change language here to be more clear?

"List all Project Id(s) you'd like to export, separated by a comma (i.e. 24,15,215). Project Id can be found in your Axe Monitor URL. 

Example:  {insert example url}"